### ISSUE TITLE
Make the back button consistent on the Notifications screen

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#421]
On empty notifications screen and normally, after receiving the notifications; the back button does not have enough margin from the left. Please make it consistent with the other screens and make sure there is invisible area of 44pxX44px.

**Comment by zhuelle** [comment]
Need prioritization.

**Comment by payeisa** [comment]
Priority added.
